feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb

```diff
@@ -0,0 +1,356 @@
+    "\n",
+    "KAFKA_BROKER = \"<EVENTHUB_HOST_NAME>:9093\"\n",
+    "KAFKA_TOPIC = \"<EVENTHUB_TOPIC>\"\n",
+    "\n",
+    "schema = AvroJsonSchema(schemaStr=\"\"\"\n",
+    "{\n",
+    "    \"type\": \"record\",\n",
+    "    \"name\": \"DriverTrips\",\n",
+    "    \"fields\": [\n",
+    "        {\"name\": \"driver_id\", \"type\": \"long\"},\n",
+    "        {\"name\": \"trips_today\", \"type\": \"int\"},\n",
+    "        {\n",
+    "            \"name\": \"datetime\",\n",
+    "            \"type\": {\"type\": \"long\", \"logicalType\": \"timestamp-micros\"}\n",
+    "        }\n",
+    "    ]\n",
+    "}\n",
+    "\"\"\")\n",
+    "\n",
+    "stream_source = KafKaSource(name=\"kafkaStreamingSource\",\n",
+    "    kafkaConfig=KafkaConfig(brokers=[KAFKA_BROKER], \n",
+    "        topics=[KAFKA_TOPIC], \n",
+    "        schema=schema\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define feature definition with the kafka source"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "driver_id = TypedKey(key_column=\"driver_id\",\n",
+    "    key_column_type=ValueType.INT64,\n",
+    "    description=\"driver_id\",\n",
+    "    full_name=\"nyc driver id\"\n",
+    ")\n",
+    "\n",
+    "kafkaAnchor = FeatureAnchor(name=\"kafkaAnchor\",\n",
+    "    source=stream_source,\n",
+    "    features=[\n",
+    "        Feature(name=\"f_modified_streaming_count\",\n",
+    "            feature_type=INT32,\n",
+    "            transform=\"trips_today + 1\",\n",
+    "            key=driver_id\n",
+    "        ),\n",
+    "        Feature(name=\"f_modified_streaming_count2\",\n",
+    "            feature_type=INT32,\n",
+    "            transform=\"trips_today + randn() * cos(trips_today)\",\n",
+    "            key=driver_id\n",
+    "        )\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build features"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client.build_features(\n",
+    "    anchor_list=[\n",
+    "        kafkaAnchor\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start streaming job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "redisSink = RedisSink(table_name=\"kafkaSampleDemoFeature\", streaming=True, streamingTimeoutMs=300000) # 5 minutes\n",
+    "settings = MaterializationSettings(name=\"kafkaSampleDemo\",\n",
+    "    sinks=[redisSink],\n",
+    "    feature_names=[\"f_modified_streaming_count\", \"f_modified_streaming_count2\"]\n",
+    ")\n",
+    "client.materialize_features(settings)\n",
+    "client.wait_job_to_finish(timeout_sec=600)"
```

Comment:
Nit: I would suggest to add a link to the Feathr documentation for this part to explain how the streaming data is loaded - i.e. whether is a long running job or it will materialise for only a period of time. Or we can add some comments here to explain what those `timeout` parameters mean.

Reply:
Based on the setting, it is a long-running job that the user would set through the `streamingTimeoutMs`.
1. `streamingTimeoutMS` parameter of `RedisSink` is the value to set on how long the spark job would ingest the kafka topics.
1. `timeout_sec` parameter of `wait_job_to_finish` method is the value on how long would the feathr client to wait until when there is no response from the spark job.

Reply:
sure thanks 🙏